File: filedata/entity/java/notice.java

package entity;
// 
public class Notice  extends BaseDomain{
    public Notice() {

    }
    
    //
    private int(255) unsigned Id;
        
    //
    private String CompanyStyle;
        
    //
    private int(255) NoticeType;
        
    //
    private String NoticeTypeDetail;
        
    //
    private String NoticeJumpUrl;
        
    //
    private int(255) unsigned Status;
        
    //
    private int(255) unsigned Orders;
        
    public int(255) unsigned getId() {
        return Id;
    }
    public void setId(int(255) unsigned Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public int(255) getNoticetype() {
        return Noticetype;
    }
    public void setNoticetype(int(255) Noticetype) {
        this.Noticetype = Noticetype;
    }
            
    public String getNoticetypedetail() {
        return Noticetypedetail;
    }
    public void setNoticetypedetail(String Noticetypedetail) {
        this.Noticetypedetail = Noticetypedetail;
    }
            
    public String getNoticejumpurl() {
        return Noticejumpurl;
    }
    public void setNoticejumpurl(String Noticejumpurl) {
        this.Noticejumpurl = Noticejumpurl;
    }
            
    public int(255) unsigned getStatus() {
        return Status;
    }
    public void setStatus(int(255) unsigned Status) {
        this.Status = Status;
    }
            
    public int(255) unsigned getOrders() {
        return Orders;
    }
    public void setOrders(int(255) unsigned Orders) {
        this.Orders = Orders;
    }
             
    @Override
    public String toString() {
        return "notice[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",NoticeType=" + NoticeType + ",NoticeTypeDetail=" + NoticeTypeDetail + ",NoticeJumpUrl=" + NoticeJumpUrl + ",Status=" + Status + ",Orders=" + Orders + "]";
    }
        
    
}
        

File: filedata/entity/c#/Notice.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("notice")]
    public class Notice
    {
        
        /// <summary>
        /// 公告ID
        /// </summary>
        [Column("Id")]
        public int Id { get; set; }
    
    
        /// <summary>
        /// 公司后缀
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 公告类型
        /// </summary>
        [Column("NoticeType")]
        public int NoticeType { get; set; }
    
    
        /// <summary>
        /// 公告内容
        /// </summary>
        [Column("NoticeTypeDetail")]
        public string NoticeTypeDetail { get; set; }
    
    
        /// <summary>
        /// 公告跳转地址
        /// </summary>
        [Column("NoticeJumpUrl")]
        public string NoticeJumpUrl { get; set; }
    
    
        /// <summary>
        /// 状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public int Status { get; set; }
    
    
        /// <summary>
        /// 排序
        /// </summary>
        [Column("Orders")]
        public int Orders { get; set; }
    
    
    }
}
